src/host/usbh_control.c

```diff
@@ -26,113 +26,60 @@
 
 #include "tusb_option.h"
 
-#if CFG_TUH_ENABLED
+#if CFG_TUH_ENABLED && 0
```

Comment:
Want to actually delete this for the PR?

Reply:
yeah, right, forgot to remove this one :)